src/Chapter2.hs

```diff
@@ -780,7 +789,7 @@ Let's now try to eta-reduce some of the functions and ensure that we
 mastered the skill of eta-reducing.
 -}
 divideTenBy :: Int -> Int
-divideTenBy x = div 10 x
+divideTenBy x = div 10 x 
```

Comment:
I see that task on eta-reduction is not finished. Do you need any help with this topic?

Reply:
Yes, I really need help. For example, I tried to eta-reduce like this `divideTenBy x = div 10`, but with that it does not compile the code :cry:

Reply:
@jacksjm No worries! When eta-reducing, you need to remove arguments from both sides of the `=` sign. So the correct solution is:

```haskell
divideTenBy = div 10
```

See how it compares with the original code:

```diff
-divideTenBy x = div 10 x
+divideTenBy = div 10
```

The motivation behind eta-reduction if that if you have two functions `foo` and `bar` of the same type, and they return the same results for any arguments (so `foo x` always equal to `bar x` for any `x`). In such situations, we say that functions `foo` and `bar` are equal, and if we want to define `foo` in terms of `bar`, we can write `foo = bar` instead of a longer `foo x = bar x` (which means that `foo` returns the same result as `bar` for any `x`).

Does it make sense?

Reply:
It makes a lot of sense! Very thanks! Now works.